Tests/FluentAssertions.Json.Shared.Specs/JTokenAssertionsSpecs.cs

```diff
@@ -662,6 +662,56 @@ public void When_jtoken_does_have_element_NotHaveElement_should_fail()
 
         #endregion (Not)HaveElement
 
+        #region HaveElementAndValue
```

Comment:
What if the subject is `null` Jtoken?

Reply:
Same as [this](https://github.com/fluentassertions/fluentassertions.json/pull/40#discussion_r461743703), no?

Reply:
Don't count on the existing methods to deal with this properly. Let's just make these correct.

Reply:
Let's resolve this thread and keep our discussion in the other one then.